docs/contributing/index.md

```diff
@@ -18,41 +18,50 @@ cd cscs-docs
 # create a branch for your changes (here we are fixing the ssh docs)
 git switch -c 'fix/ssh-alias'
 # ... make your edits ...
-# add and commit your changes
-git add <files>
-git commit -m 'update the ssh docs with aliases for all user lab vclusters'
-git push origin 'fix/ssh-alias'
 ```
-Then navigate to GitHub, and create a pull request.
 
-The `serve` script in the root path of the repository can be used to view the docs locally:
+Review your edits checking the [Guidelines](#guidelines) section below.
```

Comment:
Maybe this can be

```suggestion
Review your edits checking the [Guidelines][ref-contributing-guidelines] section below.
```
with an anchor added to the heading.